### ISSUE TITLE
Playlist search?

### ISSUE THREAD

**Issue by microvolt52** [Soundnode/soundnode-app#343]
I think it would be great if you guys would add the ability to search for playlists. Also, I can't view people's playlists. Thanks and best of luck!

**Comment by apvanzanten** [comment]
Would love this too. I listen to a lot of albums this way.

**Comment by ecupaio** [comment]
Indeed. I noticed only artist tracks are available. There's no clickthrough or tab to playlists.

**Comment by Sine1040** [comment]
Viewing users playlists would be a big plus indeed, that is the main way I use soundcloud.

**Comment by mrpinkcat** [comment]
Any news ?
